### DISCUSSION TITLE
Images not loading in emails

### DISCUSSION THREAD

**Discussion by rock7632** [dani-garcia/vaultwarden#1975] [Q&A]
I just installed the vaultwarden and did the smtp configuration but the images for github and bitwarden are not loading in the emails. Anyone have any idea? I did put my domain name in the general settings on the admin page as well. Thank you.

Edit: screenshot
![Screenshot 2021-09-14 152417](https://user-images.githubusercontent.com/14354953/133321135-e359e47e-9693-4dc6-8e15-8ed6853c3761.jpg)

**Comment by BlackDex** [comment]
Check the diagnostics page in the admin of there are any errors.
Also some mail clients block some domains or images which you need to put on an allow list.

**Reply by rock7632** [reply]
> 
> 
> Check the diagnostics page in the admin of there are any errors.
> Also some mail clients block some domains or images which you need to put on an allow list.

Thanks for your response. I checked the diagnostics page and don't see any errors in there. For the mail client, I am using Gmail as part of G Suite. Do I need to whitelist there or is that something that needs to be done within Vaultwarden?

**Reply by BlackDex** [reply]
I know Google does block some unknown domains. So it could be. But not sure. If you can access those images manually your self is either the browser or Google who blocks this.

Nothing you can do on the Vaultwarden server side of you can access those images your self.

**Reply by NoseyNick** [reply]
Works fine for me. Could you try inviting noseynick at gmail and I'll see if I can spot any misconfigurations or offer any other advice?

**Reply by rock7632** [reply]
> 
> 
> Works fine for me. Could you try inviting noseynick at gmail and I'll see if I can spot any misconfigurations or offer any other advice?

Restarted the docker container last night and after that it seems to be working fine. Thank you both for the replies though

**Reply by NoseyNick** [reply]
Cool. Was going to add this screenshot:

![Screenshot 2021-09-15 7 28 55 PM](https://user-images.githubusercontent.com/14804857/133538024-2f66be7a-427b-4549-8371-06a650a77f15.png)

... which shows gmail asking nicely if it can load my images. If I say "Display images below" or "Always display images from [my vault's email address]" it replaces the grey box with the vaultwarden logo and works fine.
